### DISCUSSION TITLE
Using Snowflake ID with PrismaORM

### DISCUSSION THREAD

**Discussion by rdrigos** [prisma/prisma#24332] [Q&A] [answered]
I'm developing a project and would like to use Snowflake ID, but I didn't find much talking about using it with Prism.
I would like to know if there is support or how to use it to create my data in the database

**Comment by ludralph** [comment]
Hi @rdrigos 👋 

We currently do not have support for this at moment. You can however follow this open [issue](https://github.com/prisma/prisma/issues/11881) and add your usecase. You can also leave a 👍 to help our engineers prioritize it.

**Reply by rdrigos** [reply]
Ok, I already left my support in the post! But is there an alternative way to use it?

**Reply by ludralph** [reply]
Hi @rdrigos 

There isn't an alternative way.

**Comment by ludralph** [comment, marked as answer]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
